caracara/common/interpolation.py

```diff
@@ -1,5 +1,12 @@
-"""
-Caracara: Variable Interpolation Logic.
+r"""Caracara: Environment Variable Interpolator.
+
+     _______                ___ ___ __          __   _____
+    |   _   .---.-.--.--.  |   Y   |  |--.---.-|  |_|__   |
+    |   1___|  _  |  |  |  |.  |   |     |  _  |   _',  ,-
+    |____   |___._|___  |  |. / \  |__|__|___._|____||--|
+    |:  1   |     |_____|  |:      |                 '--
+    |::.. . |              |::.|:. |
+    `-------'              `--- ---'
```

Comment:
Is the implication here that this class is not well named? If it's too complex, I'm happy to rethink the nomenclature.

Reply:
Nah, more that I was being fun with the concept of interpolation in general. I think the name is perfect. :-)